app/lib/katello/resources/cdn.rb

```diff
@@ -1,7 +1,18 @@
 module Katello
   module Resources
     module CDN
-      SUPPORTED_SSL_VERSIONS = ['SSLv23', 'TLSv1'].freeze
+      # the if defined? check is included for TLSv1.3 because it is not available on all platforms.
+      # at the time of writing, TLSv1 through TLSv1.2 should be available on all supported platforms,
+      # but we check regardless in case they are removed from any platform in the future.
+      # any unavailable protocols will be removed from the final Hash by the compact method.
```

Comment:
TLSv1.3 is available in our production platform, the one that end users use and I think that should aim to be a driving force. I'd agree that being able to test is important but is it the right choice for our ends users is an important question. For example, when we move to Github Actions testing or Ruby 3, this whole issue goes away. Therefore I think we should either:

 1. Default to TLSv1.3 as the newest standard
 2. Update this wording to indicate this is due to an issue with our Jenkins test environment and make sure we follow up on it when GHA or Ruby 3 happens for Katello
 
 @ekohl @evgeni y'all got any thoughts?

Reply:
Before the GHA move, we could try running Katello CI on el8 Jenkins nodes, which should also fix the problem if not available openssl-new-enough.

Reply:
Does the Red Hat CDN provide TLS 1.3? Would it cause issues with any proxy the customer is using? I think that's what determines if it can be the default.

Though I wonder about this setting at all. Why do we have it? I know it's there because in the past it needed to be changed to allow insecure corporate proxies that didn't support the latest versions, but why do we only have it for the CDN? Not for other HTTP connections? Or does it also affect custom repos?